### ISSUE TITLE
Cloudflare blocked on Happymh

### ISSUE THREAD

**Issue by DDesmond95** [keiyoushi/tachiyomi-extensions-source#5150]
### Source information

Happy h 1.4.12

### Source language

Chinese

### Steps to reproduce

1. Open any chapter using "open in webview"
2. See cloudflare block website

### Expected behavior

Should see a list of images in json

### Actual behavior

See cloudflare block website

### Mihon/Tachiyomi version

0.15.3

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Norsze** [comment]
Here you go https://keiyoushi.github.io/docs/guides/troubleshooting#cloudflare

Good luck 👍
